pkg/cmd/completion.go

```diff
@@ -42,10 +90,22 @@ func selectShell(shell string) error {
 	switch {
 	case selected == "zsh":
 		fmt.Println("Detected `zsh`, generating zsh completion file: stripe-completion.zsh")
-		return rootCmd.GenZshCompletionFile("stripe-completion.zsh")
+                err := rootCmd.GenZshCompletionFile("stripe-completion.zsh")
+                if err == nil {
+                  fmt.Println(zsh_completion_instructions)
+                }
+                return err
 	case selected == "bash":
 		fmt.Println("Detected `bash`, generating bash completion file: stripe-completion.bash")
-		return rootCmd.GenBashCompletionFile("stripe-completion.bash")
+                err := rootCmd.GenBashCompletionFile("stripe-completion.bash")
```

Comment:
The indentation here seemsto be off from the previous line. Did we run the go formatter?